package.json

```diff
@@ -11,9 +11,14 @@
     "url": "https://wordpress.org/support/plugin/facebook-for-woocommerce"
   },
   "devDependencies": {
+    "@wordpress/scripts": "^14.0.0",
     "sake": "github:skyverge/sake"
   },
   "scripts": {
-    "generate:category_attribute_json": "php bin/GenerateCategoryAttributeMapping.php"
+    "generate:category_attribute_json": "php bin/GenerateCategoryAttributeMapping.php",
+    "build:assets": "NODE_ENV=production wp-scripts build",
+    "start": "wp-scripts start",
+    "deploy:sake": "npm run build:assets && npx sake deploy --deployAssets=0",
```

Comment:
Update: I see this option is disabled in `sake.config.js`: `deployAssets: false`.

So I suspect this is fine, but there's no need to hard-code or force the option in the npm script - since it's in the config.

Reply:
> Is --deployAssets=0 needed, what does it do?

It's included in the steps listed here: https://github.com/woocommerce/facebook-for-woocommerce/wiki/Build-&-Release#deploy-via-sake

I'm not sure why this param is included though 🤔